src/organisms/cards/StaticMapCard/Readme.md

```diff
@@ -4,6 +4,7 @@ StaticMapCard Example:
     <StaticMapCard
       label='A map has no label'
       center='10010'
+      subheader='New York, NY 10011'
```

Comment:
just curious how this works in implementation? are we going to ping google for this?